### PR TITLE
test for Ellipse.evolute and Ellipse.second_moment_area added

### PR DESCRIPTION
**Pull request by faizan2700**
#### References to other Issues or PRs

Related to #16318

#### Brief description of what is fixed or changed
Increased test coverage

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by faizan2700** [comment]
I am contributing for the first time please let me know if anything needs to be changed in the PR.

File: sympy/geometry/tests/test_ellipse.py [hunk 1]
```diff
@@ -519,11 +523,26 @@ def test_auxiliary_circle():
     # a special case where Ellipse is a Circle
     assert Circle((3, 4), 8).auxiliary_circle() == Circle((3, 4), 8)
 
-
```
Comment (oscarbenjamin) [inline sympy/geometry/tests/test_ellipse.py]: This file has two blank lines between each function. Not sure why you've deleted this one. Also the functions below should have two blank lines between.

**Comment by oscarbenjamin** [comment]
Are `evolute` and `second_moment_of_area` not currently tested?

**Comment by sylee957** [comment]
It's increasing the coverage for `evolute` at least

**Comment by faizan2700** [comment]
I am sorry for late reply but I just completed my unit tests yesterday and just saw the comments. Please see the latest changes. (And what are some other libraries that will be easy for beginners for contributation, please tell me although irrelevent but I am getting tough time understanding the codebase of sympy django scrapy black and such repos, therefore I am not able to contribute to any issues.

**Comment by faizan2700** [comment]
I have made changes please tell me if more changes are to be made or please review my code. Thank you

File: sympy/geometry/tests/test_ellipse.py [hunk 2]
```diff
@@ -451,6 +451,10 @@ def test_second_moment_of_area():
     assert I_yy == e.second_moment_of_area()[1]
     assert I_xx == e.second_moment_of_area()[0]
     assert I_xy == e.second_moment_of_area()[2]
+    #checking for other point
+    t1 = e.second_moment_of_area(Point(6,5))
+    t2 = (580*pi, 845*pi, 600*pi)
+    assert t1==t2
```
Comment (oscarbenjamin) [inline sympy/geometry/tests/test_ellipse.py]: There should normally be spaces before and after `==`. See PEP8

Comment (oscarbenjamin) [reply sympy/geometry/tests/test_ellipse.py]: Also after comma e.g. `Point(6, 5)`.

**Comment by oscarbenjamin** [comment]
I left comments for future reference but I think this is good enough. Thanks!

**Pull request merged**